gulpfile.js

```diff
@@ -127,7 +144,10 @@ function syncDocsHtml(cb) {
         .on('close', cb);
 }
 
-// public tasks listed below
-
+// public tasks
+exports.compileModule = compileModule;
+exports.compileModules = compileModules;
+exports.compileBundle = compileBundle;
+exports.compileBundles = compileBundles;
```

Comment:
Why are these duplicated?

Reply:
They are all different. Can now compile an individual module or bundle, or compile all modules or bundles.

Reply:
Got it. Just seemed a bit confusing but i see the plural now.

Reply:
Can we maybe do
compileModule
compileAllModules?
That way its more intuitive than just a plural

Reply:
Yeah. Good idea.